win32/tests/sysinfo_win32_ut/CMakeLists.txt

```diff
@@ -14,4 +14,4 @@ set(${theseTestsName}_c_files
 set(${theseTestsName}_h_files
 )
 
-build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal_reals)
+build_test_artifacts(${theseTestsName} "tests/c_pal/win32" ADDITIONAL_LIBS pal_interfaces c_pal_reals pal_ll_interfaces pal_ll_win32)
```

Comment:
>pal_ll_win32 [](http://example.com/codeflow?start=120&length=12)

and remove this or replace with the c_pal_ll reals lib instead

Reply:
removed